app/controllers/admin/loan_questions_controller.rb

```diff
@@ -7,7 +7,8 @@ def index
     # Hide retired questions for now
     sets = LoanQuestionSet.where(internal_name: %w(loan_criteria loan_post_analysis)).to_a
     @json = ActiveModel::Serializer::CollectionSerializer.new(
-      sets.map { |s| s.root_group_preloaded.children_applicable_to(nil) }.flatten
+      sets.map { |s| s.root_group_preloaded.children_applicable_to(nil) }.flatten,
+      {user: current_user}
```

Comment:
Nice use of serializer options here! One thing: you don't need the curly braces since the hash is the last argument to the `new` method.